network/kademlia_test.go

```diff
@@ -370,6 +372,51 @@ func binStr(a *BzzAddr) string {
 	return pot.ToBin(a.Address())[:8]
 }
 
+func TestEachConnLB(t *testing.T) {
+	tk := newTestKademlia(t, "00000000")
+	base := tk.base
+	tk.On("00100000")
+	tk.On("00100001")
+	tk.On("00010101")
+	stats := make(map[string]int)
+	f := func(peer *Peer, po int) (bool, bool) {
+		key := hexutil.Encode(peer.Address()[:8])
+		stats[key] =  stats[key] + 1
+		// return false to only use one peer and mark it as used
+		return false, true
+	}
+	// Let n be the number of peers (3)
+	// Executing 2 * n times. We expect 2 access to each peer
+	tk.EachConnLB(base, 255, f)
+	tk.EachConnLB(base, 255, f)
+	tk.EachConnLB(base, 255, f)
+	tk.EachConnLB(base, 255, f)
+	tk.EachConnLB(base, 255, f)
+	tk.EachConnLB(base, 255, f)
+	for addr, count := range stats {
+		log.Warn("Stats", "addr", addr, "count", count)
+		if count != 2  {
+			t.Errorf("Expected 2 access to peer %v but got %v", addr, count)
+		}
+	}
+
+	stats = make(map[string]int)
+	// For each number k where m * n <= k < (m+1) *n, if we request k access,
+	// each peer should have m or m+1 access
+	k := int(rand.Int31n(200))
```

Comment:
We don't need random here. Let's just pick a number.

Reply:
Ok, I will fix a number not multiple of n